runtime/onert/frontend/circle_traininfo/src/circle_traininfo_generated.h

```diff
@@ -0,0 +1,771 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+// automatically generated by the FlatBuffers compiler, do not modify
+// command: externals/FLATBUFFER-2.0/build/flatc -c ../circle_traininfo.fbs
```

Comment:
Except for this line and copyright comments, this file is auto-generated by `flatc`.
This line is for the future, to check flatc version and options used to create this file.